### DISCUSSION TITLE
Is there any way to add new fields  to HTTP task output?

### DISCUSSION THREAD

**Discussion by sudouser777** [Netflix/conductor#2241] [Q&A]
For example see the (HTTP Task)response below
```
{
  "response": {
    "headers": {
      "Content-Type": [
        "application/json"
      ]
    },
    "reasonPhrase": "OK",
    "body": {
      "status": "success"
    },
    "statusCode": 200
  }
}
```
I want to add a new field like taskId.
```
{
  "response": {
    "headers": {
      "Content-Type": [
        "application/json"
      ]
    },
    "reasonPhrase": "OK",
    "body": {
      "status": "success",
      "taskId": "someid"
    },
    "statusCode": 200
  }
}
```
Is there anyway to achieve this(adding new fields to the task output)

**Comment by aravindanr** [comment]
I can think of two options, 

1. The `taskId` can be sent as part of the request, the HTTP task is calling and have that return the same in the response body.
2. Use a [JSON JQ task](https://netflix.github.io/conductor/configuration/systask/#json-jq-transform-task) followed by the HTTP task to transform the JSON.
